src/DFG.h

```diff
@@ -86,9 +92,11 @@ class DFG {
     void initExecLatency(map<string, int>*);
     void initPipelinedOpt(list<string>*);
     bool isMinimumAndHasNotBeenVisited(set<DFGNode*>*, map<DFGNode*, int>*, DFGNode*);
+    // target nonlinear ops
+    void nonlinear_combine();
 
   public:
-    DFG(Function&, list<Loop*>*, bool, bool, bool, map<string, int>*, list<string>*);
+    DFG(Function&, list<Loop*>*, bool, bool, bool, map<string, int>*, list<string>*, int t_vectorFactor=4);
```

Comment:
`t_vectorFactor` is only used for tuning `div`, right?

Reply:
Yes.

Reply:
Then can we rename this to `vectorFactorForIdiv` in both param.json and code?